app/services/host_fact_importer.rb

```diff
@@ -0,0 +1,40 @@
+class HostFactImporter
+  include Foreman::TelemetryHelper
+
+  attr_reader :host
+
+  def initialize(host)
+    @host = host
+  end
+
+  def create_new_record_when_facts_are_uploaded?
+    Setting[:create_new_host_when_facts_are_uploaded]
+  end
+
+  def skip_orchestration?
+    !SETTINGS[:enable_orchestration_on_fact_import]
```

Comment:
Huh, this is interesting, it seems to indicate this setting never actually worked - since `SETTINGS` contains the settings from `settings.yml` while this is actually set in the settings model which is accesible by `Setting[]`.
I suggest we drop this setting completely.

Reply:
yeah, looks like noone actually needed to do so :)
That even nihilate the only difference between `Managed` and `Base` :+1:

Reply:
I've tried to ask the original author of #4976 but I'll add another commit removing it.

Reply:
Nice find, `SETTINGS` vs `Settings` is super confusing, I propose to rename the constant to `SETTINGS_YAML` to be more explicit.

Reply:
Good point, though I'd want to avoid `YAML`. Consider `SETTINGS_CONFIG_FILES`, `CONFIG_FILE_SETTINGS` and `STATIC_SETTINGS` as alternatives.